src/main/kotlin/no/nav/sykdig/digitalisering/SykDigOppgaveService.kt

```diff
@@ -66,7 +66,7 @@ class SykDigOppgaveService(
                 sykmeldingId = UUID.randomUUID(),
                 type = UTLAND,
                 sykmelding = null,
-                endretAv = getNavEmail(),
+                endretAv = navEpost,
```

Comment:
Er litt forvirra om det er `navEpost` eller `navIdent` som er riktig. Det ser ut som det blir brukt litt om kvarandre. `endretAv` er mail her, men ser ut som ident i testane f.eks. Og så ser det ut som `tilordnetRessurs` f.eks. er navIdent. Lurer på om det er ein spesiell grunn til at det er tatt i bruk både mail og ident.

Reply:
Eg veit ikkje, Trur eg har prøvd å bruke navident der det var mulig, dette var nok berre ei refaktorering sidan begge er epost

Reply:
Kan være lurt å dobbelt sjekk om dette evt blir brukt på ein senere tidspunkt.
Eller om det bare er for sporringen sin del

Reply:
eit eksempel: Det vi lagrer i smregistrering_backend.manuelloppgave på kolonna "ferdigstilt_Av" er ein ident som Feks F123456